svc_file.go

```diff
@@ -142,6 +137,8 @@ func (c *FileService) Upload(projectID string, file FileUpload) (r FileUploadRes
 		file.CustomTranslationStatusInsertedKeys = Bool(true)
 	}
 
+	file.Queue = true
+
 	resp, err := c.post(c.Ctx(), fmt.Sprintf("%s/%s/%s/%s", pathProjects, projectID, pathFiles, "upload"), &r, file)
```

Comment:
Response code from the endpoint could be either 202 or 302 (302 is not implemented yet). Both have a `Location` header which point to an endpoint to get current upload status.

Add location and status (202 - processing, 302 - file already uploaded) to the response object

Reply:
As agreed, statuses and Location header will not be part of logic. Using response body to get process ID for final result obtain in future